plugins/topology/src/utils/resource-utils.ts

```diff
@@ -60,18 +60,18 @@ export const createOverviewItemForType = (
     case 'jobs':
       return isStandaloneJob(resource)
         ? {
-            obj: resource as K8sWorkloadResource,
+            obj: resource,
           }
         : undefined;
     case 'pods':
       return validPod(resource as V1Pod)
         ? {
-            obj: resource as K8sWorkloadResource,
+            obj: resource,
```

Comment:
@debsmita1 why is the type removed here?

Reply:
Because the type of `resource` parameter is already defined above